### PR TITLE
Add video calling for iOS

### PR DESCRIPTION
**Pull request by xumo-95**
readme for video calling iOS Quickstart

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[X] Other... Please describe:
Creating new sample
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Add Video Calling/iOSVideo/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,367 @@
+//
+//  ContentView.swift
+//  iOSVideo
+//
+//  Created by Xu Mo Microsoft on 2/23/21.
```
Comment (jsaurezlee-msft) [inline Add Video Calling/iOSVideo/ContentView.swift:5]: Should the files contain the name of the person who created them? Maybe something more generic like Created by Microsoft or Azure Communication Services?

The communication services ios hero sample has the following headers:

//
//  Copyright (c) Microsoft Corporation. All rights reserved.
//  Licensed under the MIT License.
//

**Review by raosanat — COMMENTED** [review]
https://github.com/Azure-Samples/communication-services-ios-quickstarts/pull/8/files#diff-bfa798c97f179ec0cc12d234a3b3f475d1c5e9ac974b4bd308e71ad79f1cea16

Can you remove this file ?

**Review by raosanat — COMMENTED** [review]
There is no PodFile in the project, you need to fix that.

File: Add Video Calling/iOSVideo/ContentView.swift [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+        owner = view
+        self.stream = stream
+    }
+    public func onFirstFrameRendered() {
+        let size:StreamSize = renderer!.size
+        owner.remoteVideoSize = String(size.width) + " X " + String(size.height)
+    }
+}
+
+public class CallObserver: NSObject, CallDelegate, IncomingCallDelegate {
```
Comment (raosanat) [inline Add Video Calling/iOSVideo/ContentView.swift:265]: Lets move all the Observers to one file and rename IncomingCallHandler.swift to something more generic

**Review by raosanat — CHANGES_REQUESTED** [review]
As discussed in the call lets have
1. Incoming call banner instead of accepting the call by default
2. Video On/Off should be a toggle button always available , so that user can start a call without video.

File: Add Video Calling/iOSVideo/ContentView.swift (hunk 1 above)
Comment (raosanat) [reply Add Video Calling/iOSVideo/ContentView.swift]: yes we should remove individual names

File: Add Video Calling/Podfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+platform :ios, '13.0'
+use_frameworks!
+
+target 'iOSVideo' do
+  pod 'AzureCommunicationCalling', '~> 1.0.0-beta.12'
```
Comment (raosanat) [inline Add Video Calling/Podfile:5]: Before merging lets make sure its 1.0.0 here

File: Add Video Calling/Podfile.lock [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  - AzureCommunication (1.0.0-beta.11):
+    - AzureCore (= 1.0.0-beta.11)
+  - AzureCommunicationCalling (1.0.0-beta.12):
+    - AzureCommunication (= 1.0.0-beta.11)
+  - AzureCore (1.0.0-beta.11)
+
+DEPENDENCIES:
+  - AzureCommunication (~> 1.0.0-beta.11)
+  - AzureCommunicationCalling (~> 1.0.0-beta.12)
+  - AzureCore (~> 1.0.0-beta.11)
```
Comment (raosanat) [inline Add Video Calling/Podfile.lock:11]: this is old as well ?

**Review by raosanat — APPROVED** [review]

**Pull request merged**